persistence/msmq/subscription_config_core_[,4].partial.md

```diff
@@ -1,4 +1,4 @@
-WARNING: Storing subscriptions in MSMQ is not suitable when endpoints are scaled-out, because the subscription queue cannot be shared among multiple endpoints.
+WARNING: Storing subscriptions in MSMQ **must not** be used when scaling out across multiple machines or running side-by-side on the same machine. The subscription queue cannot be shared among multiple endpoints instances for the same endpoint.
```

Comment:
I worked my way up from the bottom, so probably better to read the below comment first ;-)